roles/pulp3/tasks/install.yml

```diff
@@ -31,6 +31,7 @@
         shell: '{{ result.stdout.strip() }}'
         home: '{{ pulp_var_dir }}'
         system: true
+      when: developer_user is not defined
```

Comment:
If the developer user is defined, then it will be set to the same thing as the pulp user, since it's already created we don't want to create it again.